widget/class-widget.php

```diff
@@ -198,7 +216,7 @@ public function update( $new_instance, $old_instance ) {
       $instance = array();
 
       // Update title
-      $instance['title'] = ( ! empty( trim( $new_instance['title'] ) ) ) ? strip_tags( $new_instance['title'] ) : 'Volunteer Opportunities';
+      $instance['title'] = ( ! empty( trim( $new_instance['title'] ) ) ) ? strip_tags( $new_instance['title'] ) : '';
```

Comment:
I was using trim to make sure that the user didn't just space through the title field. If they did then ! empty will return true and the title will be the blank spaces they entered.

I've revised the code so that it no longer uses trim inside of empty.